lib/async_cache/workers/sidekiq.rb

```diff
@@ -7,8 +7,19 @@ class SidekiqWorker
       include Base
       include Sidekiq::Worker
 
-      # Only allow one job per set of arguments to ever be in the queue
-      sidekiq_options :unique => :until_executed
+      # Pulled out into a module so it can be tested.
+      module Options
+        def self.included(mod)
+          if defined?(Sidekiq::Enterprise)
+            mod.sidekiq_options unique_for: 10.minutes
```

Comment:
@tiagomjorge these behaviors are more similar than the naming implies. The Sidekiq Enterprise implementation would be equivalent to something like `until_executed_or_timeout`. The lock is released whenever the first of these two is satisfied, which is usually execution.

To make these two effectively the same we can set the `unique_for` to an arbitrarily long amount of time. 10 minutes might not be long enough for all situations though, so I agree that a documented config would be nice.